legal/refund-policy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Refund Policy
+layout: en
+no_toc: true
+---
+
+# Upgrades, Downgrades, Plan Changes
+- All monthly plan changes are automatically pro-rated and are taken into account in the next billing period.
+    - **Upgrade example:**
+You're on the 5-job plan. In the middle of your billing period, you upgrade to the 10-job plan. We will charge the rest of the month on the 10-job plan rate. We will also credit you the unused time on the 5-job plan.
+    - **Downgrade example:**
+You’re on the 10-job plan. In the middle of your billing period, you downgrade to the 5-job plan. We will charge the rest of the month on the 5-job plan rate. We will also credit you the unused time on the 10-job plan.
+- Annual plan upgrades are pro-rated and billed within 24 hours of the plan being changed
+- Annual plan downgrades are pro-rated and adjusted automatically in the next invoice
+
+# Billing Schedule
+For monthly or yearly payment plans, the Service is billed in advance on a monthly or yearly basis respectively.
+
+# Cancellation
+A user / organization can cancel their account, and their subscription will remain active for the remainder of the paid billing period.
+
+# Refunds Policy
+Upon cancellation of your account, you can request a refund under the following conditions: 
+- **Request made up to and including 14 days after the billing date:**
+
+  * Monthly plans: full refund
+  * Annual plans: full refund
+
+- **Request made between 15 and 30 days after the billing date:**
+
+  * Monthly plans: no refund
+  * Annual plans: first month billed at the monthly plan rate, and remainder is refunded
+
+- **Request made 31 days or more after the billing date:**
+
+   * Monthly plan: not applicable
+   * Annual plans: no refund for cancellations made on or after the 31st day of the subscription
+
+## Refunds across the Fiscal Year
+In this case, a refund is requested that would involve payments made in another tax year, ie for an Annual plan that ran from October to October. Once the fiscal year is closed, we are not legally allowed to go back into the previous year. In Germany, the financial and tax year follows the calendar year - that is, the financial year begins on January 1 and ends on December 31.
```

Comment:
Here is looks like we're starting with the example, and it isn't too clear. People might not know or care that we are a German company. 

What about something like

> Once the fiscal year is closed, we are not legally allowed to issue refunds from the previous year. As a German company, Travis CI considers the financial and tax year to mactch the calendar year - that is, the financial year begins on January 1 and ends on December 31.

Example: If your annual plan runs from Octo....